src/backend/mock.ts

```diff
@@ -168,3 +147,43 @@ export default class MockBackend extends events.EventEmitter implements Kubernet
   }
   // #endregion
 }
+
+class MockKubernetesBackend extends events.EventEmitter implements KubernetesBackend {
```

Comment:
From what I can tell, you're missing `KubernetesBackendPortForwarder` and `EventEmitter` in the `implements` part.

Reply:
`KubernetesBackend` extends those two, so it's implied. (Otherwise this wouldn't properly implement `KubernetesBackend`, so it would be a compile error.)

Reply:
Ah I see, I missed that. :+1: